js/main.js

```diff
@@ -245,6 +245,8 @@ const MM = (function () {
 		if (moduleWrapper !== null) {
 			moduleWrapper.style.transition = "opacity " + speed / 1000 + "s";
 			moduleWrapper.style.opacity = 0;
+			moduleWrapper.className = moduleWrapper.className.split(" shown").join("");
+			moduleWrapper.className += " hidden";
```

Comment:
I think it might be better to use `classList.add("hidden")` and `classList.remove("hidden")` in stead of doing string manipulation.

Reply:
did not know this functions. Changed to them now